draft-ietf-rats-corim.md

```diff
@@ -1523,44 +1535,44 @@ To facilitate the description of normative Verifier behavior, this document uses
 The following terms are used:
 
 {: vspace="0"}
-Claim:
+Property:
 : A piece of information, in the form of a key-value pair.
```

Comment:
Here the Property and the Claim definition that RFC9334 inherits collide a bit. Maybe something like:

```suggestion
: A piece of information, in the form of a Claim as specified in {{-rats-arch}} where the name of the name/value pair is always an integer with an annotated text string name.
```

Reply:
```
: Compare: Claim {{-rats-arch}}
```